code/game/gamemodes/objective.dm

```diff
@@ -136,6 +136,8 @@ GLOBAL_LIST_EMPTY(objectives) //SKYRAT EDIT ADDITION
 			continue
 		if(possible_target in blacklist)
 			continue
+		if(istype(target_area, /area/centcom/interlink)) //SKYRAT EDIT: people in the interlink can't be objective targets
```

Comment:
This is somewhat dangerous, because there's a specific station trait that makes everyone spawn on the interlink. I'm sure there's a check to see if the round was started that you could add before that so it doesn't affect roundstart, but affects everything else.

Reply:
Pushed a change that should fix it